plugins/volumeFilter.go

```diff
@@ -228,6 +241,17 @@ func volumeFilterFn(dailyOTB *VolumeFilterConfig, dailyTBBAccumulator *VolumeFil
 	return nil, nil
 }
 
+func getRemainingCapacity(otbCap float64, tbbCap float64, limitCap float64, amount float64, price float64) float64 {
```

Comment:
This function is a direct implementation of the previous capacity computations ([base](https://github.com/stellar/kelp/pull/602/files#diff-21b706ce390d2571295650103c3c1953c7d81665c8b2672c9de2208dc9541952L182-L192), [quote](https://github.com/stellar/kelp/pull/602/files#diff-21b706ce390d2571295650103c3c1953c7d81665c8b2672c9de2208dc9541952L200-L210)).

(Note that when invoked above, in the base if-statement, the price is passed as 1 - thus making this implementation generalizable in either case.)